bescort.lic

```diff
@@ -448,6 +455,58 @@ class Bescort
     end
   end
 
+  def take_sandbarge(mode, mode_2)
+    case mode
+    when 'muspari'
+	  if mode_2 == 'oasis'
```

Comment:
Have you considered two `case` blocks, one for each arg, instead of nested conditionals? I think that might make the code a bit clearer in its intent.